internal/characters/xiao/plunge.go

```diff
@@ -65,7 +65,7 @@ func (c *char) HighPlungeAttack(p map[string]int) action.ActionInfo {
 		Durability: 25,
 		Mult:       highplunge[c.TalentLvlAttack()],
 	}
-	c.Core.QueueAttack(ai, combat.NewCircleHit(c.Core.Combat.Player(), 2), highPlungeHitmark, highPlungeHitmark, c.c6cb())
+	c.Core.QueueAttack(ai, combat.NewCircleHit(c.Core.Combat.Player(), 6), highPlungeHitmark, highPlungeHitmark, c.c6cb())
```

Comment:
needs burst status check, should be radius 5 outside of burst (this is relevant because xiao's burst can expire mid plunge afaik)